integration-tests/integration.scala

```diff
@@ -14,9 +14,9 @@ val javaVersion                 = Config.DefaultJavaVersion
 val scalaVersion                = Config.DefaultScalaVersion
 val coreVersion                 = os.read(os.pwd / "version.txt").trim
 val scalaPluginVersion          = coreVersion
-val providerRandomSchemaVersion = "4.14.0"
-val providerTlsSchemaVersion    = "4.11.1"
-val providerPurrlSchemaVersion  = "0.4.1"
+val providerRandomSchemaVersion = "4.15.1"
+val providerTlsSchemaVersion    = "5.0.0"
+val providerPurrlSchemaVersion  = "0.5.0"
```

Comment:
we're forced to bump this every time now?

Reply:
yes, I'll reuse `scripts/Version` in the future, so version update will be automatic